src/pages/learn/index.tsx

```diff
@@ -0,0 +1,929 @@
+
+  .featured {
+    padding-left: 1rem;
+    margin-left: -1rem;
+    border-left: 1px dotted ${(props) => props.theme.colors.primary};
+  }
+
+  .citation {
+    p {
+      color: ${(props) => props.theme.colors.text200};
+    }
+  }
+`
+const Card = styled(OriginalCard)`
+  justify-content: space-between;
+  h3 {
+    margin-top: 0;
+  }
+`
+
+const CardGradient = styled(Card)`
+  justify-content: start;
+  background: ${(props) => props.theme.colors.cardGradient};
+
+  ul {
+    flex: 1;
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: center;
+    margin-bottom: 0;
+  }
+`
+
+const CardImage = styled.div`
+  text-align: center;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-bottom: 1rem;
+`
+
+const DocsContainer = styled.div`
+  margin: 0 9rem;
+  display: flex;
+  flex-flow: column nowrap;
+  gap: 0.8rem;
+
+  @media (max-width: ${(props) => props.theme.breakpoints.xl}) {
+    margin: 0;
+  }
+`
+
+const AdditionalReadingHeader = styled.h3`
+  margin-top: 4rem;
+  font-size: 1.2rem;
+  font-weight: 700;
+  text-align: center;
+`
+
+const Banner = styled.div`
+  margin: 3rem 0;
+  display: flex;
+  border-radius: 10px;
+  overflow: hidden;
+  background: ${(props) => props.theme.colors.cardGradient};
+
+  h3 {
+    margin-top: 0;
+  }
+
+  ul {
+    margin-bottom: 0;
+  }
+
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    flex-flow: column nowrap;
+  }
+`
+
+const BannerBody = styled.div`
+  padding: 3rem;
+`
+
+const BannerImage = styled.div`
+  align-self: end;
+`
+
+const Section = styled.section`
+  margin-top: 6rem;
+  &:first-child {
+    margin-top: 0;
+  }
+`
+
+const CardGrid = styled(OriginalCardGrid)`
+  margin-top: 2rem;
+`
+
+const LearnPage = ({ data }: PageProps<Queries.LearnPageQuery, Context>) => {
+  const intl = useIntl()
+  const isRightToLeft = isLangRightToLeft(intl.locale as Lang)
+
+  const tocItems = [
+    {
+      id: "what-is-crypto-ethereum",
+      title: "What is crypto and Ethereum?",
+    },
+    {
+      id: "how-do-i-use-ethereum",
+      title: "How do I use Ethereum?",
+    },
+    {
+      id: "what-is-ethereum-used-for",
+      title: "What is Ethereum used for?",
+    },
+    {
+      id: "strengthen-the-ethereum-network",
+      title: "Strengthen the Ethereum network",
+    },
+    {
+      id: "learn-about-ethereum-the-protocol",
+      title: "Learn about Ethereum the protocol",
+    },
+    {
+      id: "learn-about-the-ethereum-community",
+      title: "Learn about the Ethereum community",
+    },
+  ]
+
+  const heroContent = {
+    title: "Learn Hub",
+    header: "Learn about Ethereum",
+    subtitle:
+      "Your educational guide to the world of Ethereum. Learn how Ethereum works and how to connect to it. This page includes technical and non-technical articles, guides, and resources.",
+    image: getImage(data.heroImage),
+    alt: "Illustration of a group of people marvelling at an ether (ETH) glyph in awe.",
```

Comment:
Interesting. Curious about why is it like that? is the reader going to tell some generic image description to the user?

Reply:
@pettinarip screen readers will completely ignore empty alts (i.e. alt="") because it treats them as decorative. You can imagine, as a screen reader user, that hearing descriptions of decorative images hundreds of times per day would get pretty annoying 😝

[Here is the W3 guidelines on decorative images.](https://www.w3.org/WAI/tutorials/images/decorative/)